docs/src/main/sphinx/release/release-411.md

```diff
@@ -0,0 +1,59 @@
+# Release 411 (15 Mar 2023)
+
+## General
+
+* Add spilled data size to query statistics. ({issue}`16442`)
+* Add {func}`sinh` function. ({issue}`16494`)
+* Add support for a `GRACE PERIOD` clause in the `CREATE MATERIALIZED VIEW`
+  task. This is a backwards incompatible change, and the previous behavior can
+  temporarily be restored with the `legacy.materialized-view-grace-period`
+  configuration property or the `legacy_materialized_view_grace_period` session
+  property. ({issue}`15842`)
+* Add `CANCELING`, `ABORTING`, and `FAILING` task statuses for tasks which are
```

Comment:
CANCELLING ??!!

Reply:
I also looked this up, and canceling _is_ considered correct in American English? Which is news to me?

Reply:
i am not convinced we need this. as a user, i wouldn't understand what is this about